### DISCUSSION TITLE
Segment-Anything v0.7.0

### DISCUSSION THREAD

**Discussion by kadirnar** [kadirnar/segment-anything-video#59] [Announcements]
## What's Changed
* add video support for manuel_mask by @kadirnar in https://github.com/kadirnar/segment-anything-video/pull/50
* add falai(server) support by @kadirnar in https://github.com/kadirnar/segment-anything-video/pull/56
* add manuel mask for server by @kadirnar in https://github.com/kadirnar/segment-anything-video/pull/57
* update version by @kadirnar in https://github.com/kadirnar/segment-anything-video/pull/58

**Full Changelog**: https://github.com/kadirnar/segment-anything-video/compare/v0.6.0...v0.7.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kadirnar/segment-anything-video/releases/tag/v0.7.0'>Segment-Anything v0.7.0</a>.</em>

**Comment by G-force78** [comment]
How to use manual mask selector in gradio app?
